CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to @dhl-uu/timing-util
+
+## Required software
+
+- Node.js
+- npm
+- git
+- [editorconfig plugin][editorconfig] for your favorite editor
+
+[editorconfig]: https://editorconfig.org/
+
```

Comment:
As a general comment,  I would say "required software" should always include a link to the software.